code/__DEFINES/misc.dm

```diff
@@ -185,10 +185,15 @@ GLOBAL_LIST_EMPTY(bloody_footprints_cache)
 //subtypesof(), typesof() without the parent path
 #define subtypesof(typepath) ( typesof(typepath) - typepath )
 
-//Gets the turf this atom inhabits
+/// Get the turf that `A` resides in, regardless of any containers.
+///
+/// Use in favor of `A.loc` or `src.loc` so that things work correctly when
+/// stored inside an inventory, locker, or other container.
 #define get_turf(A) (get_step(A, 0))
 
-//Same as above except gets the area instead
+/// Get the ultimate area of `A`, similarly to [get_turf].
+///
+/// Use instead of `A.loc.loc`.
```

Comment:
Does not follow code doc guidelines, see https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md